app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+
+  after_save :update_like_counter
+
+  def update_like_counter
+    post.increment!(:LikesCounter)
+  end
```

Comment:
- [ ] OPTIONAL
Following best practices `update_like_counter` should be marked as private after implementing the `after_save` method.